stylesheets/styles.css

```diff
@@ -0,0 +1,49 @@
+html, body {
+    height: 100%;
+}
+
+body {
+    font-family: Georgia, 'Times New Roman', Times, serif;
+    background: rgb(175,171,241);
+    background: linear-gradient(0deg,rgb(227, 172, 249,1) 0%, rgba(120,40,235,1) 35%, rgb(227, 172, 249,1) 100%);
+    display: flex;
+    flex-direction: column;
+}
+
+h2 {
+    text-align: center;
+    color: rgb(250, 218, 157);
+    font-weight: bold;
+}
+
+
+#main-menu {
```

Comment:
Nice use of the overall styles file to set up `#main-menu`'s styling across the pages.